task/s2i-java/0.1/s2i-java.yaml

```diff
@@ -143,7 +145,7 @@ spec:
       name: varlibcontainers
     securityContext:
       runAsUser: 0
-  - image: registry.redhat.io/ubi8/python-39:1-90.1669637098
+  - image: registry.redhat.io/ubi8/python-39:1-87.1669838026
```

Comment:
@Michkov What is this changed for?

Reply:
I put there wrong unexisting tag in PR before so this was a fix